README.md

```diff
@@ -1,4 +1,4 @@
-# React Native Test App
+# React Native Test App 
```

Comment:
You managed to add a space at the end when you removed the newline. 😂

Reply:
Cool.. also looks like the CI build cannot handle builds from forks.

Reply:
Not sure what you mean. One of the tests are failing because we're including more files than expected: https://github.com/microsoft/react-native-test-app/actions/runs/8882235888/job/24386416320#step:11:1

Reply:
Do we need all the extra files?